### PR TITLE
added a busy loop into all the code and some comments. We're done wit…

### PR DESCRIPTION
**Pull request by shem-snow**
We did the thing. Our code has busy loops in it.

### CODE REVIEW

**Comment by SamBosch77** [comment]
Keep in mind that item 11 under measuring jitter asks to calculate the expected drift over a one-hour period. I didn't see any calculations for that in the README, so you may want to add that

File: src/gpio_interrupt.c
Comment (VarunBhavya) [inline src/gpio_interrupt.c:1]: It appears that the `busy_loop` function is defined, yet I don't see it being utilized within the `irq_callback` function. Instead, the code implements a sleep for 36 ms. Could you clarify if this is an oversight, or is there a specific reason for not using the `busy_loop` function in this context?

**Review by VarunBhavya — CHANGES_REQUESTED** [review]
I’ve noticed that in all files, there is an incrementing loop that continues until a count of 36 is reached. However, there are no comments explaining the reasoning behind this specific choice of 36. Could you provide some context on why this value was selected? Understanding this will help me before approving the pull request.

File: src/gpio_interrupt.c
Comment (shem-snow) [reply src/gpio_interrupt.c:1]: We didn't realize there was already a busy_loop function and made our own. The amount of time is not relavant.

**Pull request merged**